styleguide/derek/pattern-components/resourceCenter-resource/refresh/_resourceCenter-resource.scss

```diff
@@ -0,0 +1,141 @@
+// Resource Icon Color
+@mixin resource-refresh($class, $color) {
+  #{$class} {
+    @include make-md-column(4);
+    @include make-sm-column(6);
+    background: $white;
+    box-shadow: 0 1px 1px 1px rgba(0, 0, 0, .06);
+    padding: 0;
+    transition: .3s ease-in-out;
+    width: 100%;
+
+    .resource-top {
+      background: $color;
+    }
+
+    a {
```

Comment:
We should use a class for here if possible.

Reply:
We tried this with both iterations of the resource cards so far and removing it breaks all the styling. I just tried it again to confirm and unfortunately that is still the case.